.github/workflows/ci-nym-vpn-core.yml

```diff
@@ -202,7 +202,8 @@ jobs:
         if: contains(matrix.os, 'macos') || contains(matrix.os, 'mac-m1')
         working-directory: nym-vpn-core
         run: |
-          cargo build --verbose --target aarch64-apple-ios -p nym-vpn-lib
+          rustflags="-L ${GITHUB_WORKSPACE}/build/lib/aarch64-apple-ios"
+          RUSTFLAGS=$rustflags cargo build --verbose --target aarch64-apple-ios -p nym-vpn-lib
```

Comment:
General comment _not_ for this PR:
The platform stuff is really reaching a point where we should try to take a step back and rework it. Also I'm kinda think that this part here is the result of building both mac and ios during a single job while splitting them into separate seems likely more appropriate (like how android is separate from linux).

Reply:
Yeah that's legit. Let's add a card for this and attack it in a separate PR.